hwconf/hw_rbat.h

```diff
@@ -17,10 +18,11 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
     */
 
-#ifndef HWCONF_HW_RBAT_V1_H_
-#define HWCONF_HW_RBAT_V1_H_
+#ifndef HWCONF_HW_RBAT_H_
+#define HWCONF_HW_RBAT_H_
 
-#define HW_NAME					"rbat_v2"
+#define HW_NAME					"rbat"
+#define HW_REV_CHAR				'A'
 
```

Comment:
Sorry for the flaw here but should be 'B' here since this is the target revision as from now on.